server/views/partials/footer.ejs

```diff
@@ -0,0 +1,13 @@
+  
+ <!-- 
+Author: Multiverse Design
+Date: Nov-10-2021
+FileName : footer.js --> 
+  
```

Comment:
place her nav footer section <!--footer-->

Reply:
I have transferred from bottom_nav to footer section